pupa/tests/importers/test_event_importer.py

```diff
@@ -1,8 +1,7 @@
 import pytest
 from pupa.scrape import Event as ScrapeEvent
-from pupa.importers import EventImporter
-from opencivicdata.models import Jurisdiction
-
+from pupa.importers import EventImporter, OrganizationImporter, PersonImporter, BillImporter, VoteEventImporter
```

Comment:
The vote event importer test may be a good format to copy, as I had to write similar tests when updating it to reconcile voters.